src/components/vr_cooperation/CMakeLists.txt

```diff
@@ -53,9 +54,15 @@ set (LIBRARIES
     ApplicationManager
     Utils
     ConfigProfile
+    protobuf
+    HMIProto
 )
 
+generate_proto(${CMAKE_CURRENT_SOURCE_DIR}/interface/hmi.proto HmiProto)
```

Comment:
`generate_proto(${CMAKE_CURRENT_SOURCE_DIR}/interface/hmi.proto HMIProto)`
HmiProto -> HMIProto, because you set SOURCE "HMIProto"